.github/ISSUE_TEMPLATE/chore.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Chore
+about: Suggest a chore that will help contributors and/or developers
```

Comment:
```suggestion
about: Suggest a chore that will help contributors and doesn't affect end users.
```

IMO contributors already encompasses developers. 

Also, I'd like to clarify that this type of issue should not impact end users.